recipes/syngap/meta.yaml

```diff
@@ -1,13 +1,13 @@
 {% set name = "syngap" %}
-{% set version = "1.2.2" %}
+{% set version = "1.2.1" %}
```

Comment:
This is actually a downgrade from 1.2.2 to 1.2.1.
Is this what you want ?

Reply:
Yes. 1.2.2 was a test version for Azure CL but so how it was merged throught https://github.com/bioconda/bioconda-recipes/pull/47479#event-13148505956 in a wrong build way. And it may cause running issue for users.

Reply:
> This is actually a downgrade from 1.2.2 to 1.2.1. Is this what you want ?
Yes. 1.2.2 was a test version for Azure CL but so how it was merged throught https://github.com/bioconda/bioconda-recipes/pull/47479#event-13148505956 in a wrong build way. And it may cause running issue for users.

Reply:
> This is actually a downgrade from 1.2.2 to 1.2.1. Is this what you want ?

Is it allowed to do this?

Reply:
> This is actually a downgrade from 1.2.2 to 1.2.1. Is this what you want ?

Yes. 1.2.2 was a test version for Azure CL but so how it was merged throught https://github.com/bioconda/bioconda-recipes/pull/47479#event-13148505956 in a wrong build way. And it may cause running issue for users.

Reply:
In that case it is better to leave `1.2.2` as a version and increase the `build > number`.
When a user do `conda install xyz` the logic will install the latest version at Anaconda.org, i.e. 1.2.2, not the latest/current in the Git repo.

Reply:
> In that case it is better to leave `1.2.2` as a version and increase the `build > number`. When a user do `conda install xyz` the logic will install the latest version at Anaconda.org, i.e. 1.2.2, not the latest/current in the Git repo.

Got it. I'll try. Thanks!

Reply:
> But if 1.2.2 was a "mistake" then re-release 1.2.1 as 1.2.3

OK